lib/vagrant-openshift/hooks/windows_wait.rb

```diff
@@ -0,0 +1,38 @@
+#--
+# Copyright 2016 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#++
+
+require_relative "../aws"
+
+module Vagrant
+  module Openshift
+    module Hooks
+      class WindowsWait
+        def initialize(app, env)
+          @app = app
+        end
+
+        def call(env)
+          if env[:machine].config.openshift.autoconfigure_aws == true && env[:machine].config.ssh.username == "Administrator"
```

Comment:
Since the `env[:machine].config.ssh.username == "Administrator"` block happens often, I think it would be nice to see it as a explanatory helper function:
```rb
def is_windows?(machine)
  return machine.config.ssh.username == "Administrator"
end
```